### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by stdavis**
## Description of Changes
This pull requests cleans up some things that stood out to me after reviewing the changes made yesterday. I could be way out in left field but I don't believe that they changes to anything but get rid of unnecessary code.

We may want to trash `scratch.gdb` when this PR is merged since it will have a bunch of old temp tables that will never get deleted.

### Test results and coverage

```
Name                     Stmts   Miss     Cover   Missing
---------------------------------------------------------
forklift\arcgis.py          65      3    95.38%   48, 96-98
forklift\cli.py            224     64    71.43%   138, 173-178, 185, 232-235, 283-300, 307-318, 328-377
forklift\core.py           226      8    96.46%   84-88, 155, 199, 343, 418
forklift\lift.py           128     26    79.69%   44-45, 131-152, 158, 176-177, 213-215, 228-229
forklift\models.py         168      4    97.62%   75, 317, 352-353
---------------------------------------------------------
TOTAL                      900    105    88.33%

5 files skipped due to complete coverage.
-----------------------------------------------------------------------------
149 tests run in 338.334 seconds (149 tests passed)
```

### Speed test results

```
Dry Run: 5.23 minutes
Repeat: 1.68 minutes
```

### CODE REVIEW

**Comment by steveoh** [comment]
> We may want to trash scratch.gdb when this PR is merged since it will have a bunch of old temp tables that will never get deleted.

Possible enhancement for [the internal optimize method](https://github.com/agrc/forklift/blob/e46d9d5a2630af30b4ea64704324b96941a3d3b9/src/forklift/core.py#L64-L66)? Instead of compacting, should we delete it?

**Comment by stdavis** [comment]
> Possible enhancement for the internal optimize method? Instead of compacting, should we delete it?

Yes, I think that's probably faster/easier/cleaner to just delete it.

**Comment by stdavis** [comment]
Don't merge this quite yet. I just pushed these new commits so that I could test them out on the server...

File: src/forklift/core.py [hunk 1]
```diff
@@ -281,12 +273,23 @@ def _hash(crate, hash_path, needs_reproject):
     insert_cursor = arcpy.da.InsertCursor(changes.table, changes.fields)
 
     with arcpy.da.SearchCursor(crate.source, fields, sql_clause=sql_clause) as cursor:
+        def parse_id_as_int(id):
+            return str(int(id))
+
+        def parse_id_as_string(id):
+            return str(id)
+
+        if crate.source_primary_key_type == int:
```
Comment (steveoh) [inline src/forklift/core.py]: shouldn't this be float or some other type

Comment (stdavis) [inline src/forklift/core.py]: No, I didn't get that specific. I just classify as int or str.

Comment (steveoh) [inline src/forklift/core.py]: ok it is confusing that if it's an int your casting it to int again. We might add a comment to this to make that more clear.

**Comment by stdavis** [comment]
OK. I think that this is ready for review again.

Updated stats after adding last two commits:
### Test results and coverage

```
lint: commands succeeded
congratulations :)

Name                     Stmts   Miss     Cover   Missing
---------------------------------------------------------
forklift\arcgis.py          65      3    95.38%   48, 96-98
forklift\cli.py            224     64    71.43%   138, 173-178, 185, 232-235, 283-300, 307-318, 328-377
forklift\core.py           233      8    96.57%   84-88, 155, 199, 354, 429
forklift\lift.py           128     26    79.69%   44-45, 131-152, 158, 176-177, 213-215, 228-229
forklift\models.py         168      4    97.62%   75, 317, 352-353
---------------------------------------------------------
TOTAL                      907    105    88.42%

5 files skipped due to complete coverage.
-----------------------------------------------------------------------------
150 tests run in 392.518 seconds (150 tests passed)
```

### Speed test results

```
Dry Run: 5.6 minutes
Repeat: 1.83 minutes
```

**Pull request merged**